### PR TITLE
[Added] digits classification (LR)

### PR DESCRIPTION
**Pull request by Rohit-bisht-rise**
Here is the Commit for digits classification using Logistic Regression.
It has done in the following manner
1. load data from sklearn.dataset
2. converted to pandas datafarame
3. check dtype and missing values
4. Normalized and split into test and train
5. train the model & prediction 
6. score with f1
7. classification report 
8. confusion matric

### CODE REVIEW

**Comment by Rohit-bisht-rise** [comment]
#85 this is the issued @Jayshah6699
I was facing a problem to clone this repo so I forked it and added the file directly to the GitHub ( main directory ), if everything is ok then merge it into the main branch and move it into a new Folder name: DigitRecogination(LR)

**Comment by Jayshah6699** [comment]
no delete that and add it in different folder

**Comment by Jayshah6699** [comment]
You need to add your work in handwritten digit recognition folder only. @Rohit-bisht-rise
